cgmanifest.json

```diff
@@ -2190,6 +2190,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libacvp",
```

Comment:
Please also update the licenses map file under `SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`. I know, I know, we need some automation for all of these requirements...;)

Reply:
Would this go under the Microsoft MIT license?

Reply:
Yup, that's correct.